lib/public_key/src/pubkey_cert.erl

```diff
@@ -1362,6 +1369,20 @@ public_key(#'DSAPrivateKey'{p=P, q=Q, g=G, y=Y}, _) ->
     Algo = #'PublicKeyAlgorithm'{algorithm= ?'id-dsa', 
 				 parameters={params, #'Dss-Parms'{p=P, q=Q, g=G}}},
     #'OTPSubjectPublicKeyInfo'{algorithm = Algo, subjectPublicKey = Y};
+public_key(#'ECPrivateKey'{version = _Version,
+			  privateKey = _PrivKey,
```

Comment:
strange indentation for privateKey here and some lines below

Reply:
Humm .. wonder how that happened, I will fix this.